script.js

```diff
@@ -89,6 +89,13 @@ for (let i = 0; i < designers.length; i++) {
     const fashionImages = document.createElement('div');
     fashionImages.classList.add('fashion-images');
 
+    const chess = document.createElement('img');
+    chess.classList.add('chess-img');
+    chess.src = '/images/chess.jpg';
+    chess.alt = 'chess photo';
```

Comment:
- [ ] Please your `chess.jpg` is not showing on the page as you can see in the video below and I think the issue is you forgot to bring `.` below `/images/chess.jpg` so changing ` chess.src = '/images/chess.jpg';` to  `chess.src = './images/chess.jpg';` will fix the issue :+1: 

[Screencast from 03-21-2023 09:07:53 PM.webm](https://user-images.githubusercontent.com/92303891/226742304-f7edff2e-82a4-4d59-a2fd-3ba801deed48.webm)